docs/getting-started/openapi.md

```diff
@@ -4,4 +4,5 @@ Spectral is a generic linter, but a lot of effort has been put in to making sure
 
 Run Spectral against a document without specifying a ruleset will trigger an auto-detect, where Spectral will look to see if `swagger: 2.0` or `openapi: 3.0.x` is at the top of the file. If it finds either of those it will load `spectral:oas2` or `spectral:oas3`, both of which are documented in our [Reference > OpenAPI Rules](../reference/openapi-rules.md).
 
-_**Note:** If you would like support for other API description formats like RAML, message formats like JSON:API, or anything at all, we recommend you start building up custom, but generic rulesets that can be shared with others. These can then be shared around in a sort of marketplace in the future._
+<!-- theme: info -->
+> If you would like support for other API description formats like [RAML](https://raml.org/), message formats like [JSON:API](https://jsonapi.org/), event-driven API descriptions like [AsyncAPI](https://asyncapi.io/), etc., we recommend you start building custom but generic rulesets which can be shared with others. We've started putting together some over here on [OpenAPI Contrib](https://github.com/openapi-contrib/style-guides/)._
```

Comment:
:heart:

Reply:
I like this very much -> https://github.com/openapi-contrib/style-guides/ <3

Reply:
@nulltoken thanks! Can we get a +1? Merging to master needs two thumbs.